src/test/resources/MakeAppointment.feature

```diff
@@ -0,0 +1,37 @@
+Feature: MakeAppointment
+  An a user
+  I want to sign in
+  I want to make appointment and confirmation appointment
+
```

Comment:
sửa thành
An a user
I want to make  an appointment 
So I can make an appointment with the doctor